plugin/ipythonPlugins/src/dist/python3/python3Plugin

```diff
@@ -21,7 +21,7 @@ import subprocess
 if len(sys.argv) > 1:
     arg1 = sys.argv[1]
     if arg1 == "--version":
-        os.execlp("ipython", "ipython", "--version")
+        os.execlp("ipython", "ipython", "-c", "\"import pip; import IPython; print( \'ipywidgets==\' + \'%s\' % sorted(filter(lambda x : x.key == \'ipywidgets\', pip.get_installed_distributions()))[0].version ) ; print(\'ipython==\' + str(IPython.version_info[0])  + '.' + str(IPython.version_info[1]) + '.' + str(IPython.version_info[2]))\"");
```

Comment:
I was not succeeded in "ipywidgets.__version__"
This code is not last version, 
Last one like :

arr = list(filter(lambda x : x.key == \'ipywidgets\', pip.get_installed_distributions())); print (\'ipywidgets==%s\' % arr[0].version) if len(arr) > 0 else \'NONE\';